e2e/bitrise.yml

```
@@ -0,0 +1,193 @@
+format_version: "11"
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+workflows:
+
+  test_nested_module:
+    title: Test simple Android project with nested module
+    envs:
+    - SAMPLE_APP_URL: https://github.com/bitrise-io/android-multiple-test-results-sample.git
```

```suggestion
    - TEST_APP_URL: https://github.com/bitrise-io/android-multiple-test-results-sample.git
```
As used for example in https://github.com/bitrise-steplib/steps-xcode-test/blob/5de2593fa7291ff3e6bf25b54f061da7ee823713/e2e/bitrise.yml.

---

Thank you, fixed. (I didn't accept this suggestion because there are more instances of this env var in the file)